docs/reference/contributing/target/flash.md

```diff
@@ -57,6 +59,10 @@ Bootloader-ready declaration of flash VTOR address:
 #endif
 ```
 
+#### Targets.json Metadata
+
+The managed and un-managed bootloader builds require some target metadata from CMSIS PACKs. Add a `device_name` attribute to your target as described in [Adding and Configuring Targets](https://os.mbed.com/docs/v5.6/tools/adding-and-configuring-targets.html).
```

Comment:
- Please delete the hyphen in "unmanaged".
- Please change "PACKs" to "Packs" for consistency across documents.
- Please change "as described in Adding and Configuring Targets" to "as Adding and configuring targets describes" for active voice.
- Please change the link to look like this: 
```<a href="https://os.mbed.com/docs/v5.6/tools/adding-and-configuring-targets.html" target="_blank">Adding and configuring targets</a>```

Reply:
Ah, spell check did not like unmanaged probably.

Reply:
That's probably it. It's bad sometimes.